execute/test-execute.sh

```diff
@@ -483,6 +483,7 @@ test_exec_stdio_file() {
     [[ $? -eq 0 ]]
 }
 
+test_exec_dynamic_user
```

Comment:
Please describe in the commit message why you moved it up.

Reply:
Hi @ssahani ,

1. I investigated, that current changes works for me under classic and atomic host for F26, F27, Rawhide.

2. Actually I haven't enough time for deep debugginh, but I noticed, that it works for me, if it "test_exec_dynamic_user" case move at the beginning of the list. Does this the case, when it should pass after execution of group of exec_* tests? 

Thanks,
Eduard